src/libraries/Microsoft.Bcl.Cryptography/src/System/Security/Cryptography/NetStandardShims.cs

```diff
@@ -114,6 +114,7 @@ internal static bool TryGetHashAndReset(
         }
     }
 
+#if !NETSTANDARD2_1_OR_GREATER
```

Comment:
I don't see this being used - just making sure this is needed.

Reply:
`CryptographicOperations` is used in M.B.Cryptography in a few places such as 

https://github.com/dotnet/runtime/blob/fa004fb5ce5ec9f99d1c3ba3adc48c9473cc8eaa/src/libraries/Microsoft.Bcl.Cryptography/src/System/Security/Cryptography/SP800108HmacCounterKdfImplementationManaged.cs#L54

Since we are adding a .NET Standard 2.1 target, and `CryptographicOperations` is part of .NET Standard 2.1, we need to omit it of the target is .NET Standard 2.1 compatible since it will just use whatever the .NET Standard implementation is.

Reply:
CryptographicOperations is undoubtedly used deep in the guts of PasswordBasedEncryption.  The type already exists in netstandard2.1, so now that we're also building for netstandard2.1 (when we used to not) it needs to not redefine it.